spec/MacAddressTableRecorder.yaml

```diff
@@ -2757,27 +2902,27 @@ paths:
                   type: string
                   description: >
                     'Name of application that shall record the OaM requests
-                    find [/core-model-1-4:control-construct/logical-termination-point=matr-1-0-0-http-c-ol-2-0-1-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/application-name]'
+                    find [/core-model-1-4:control-construct/logical-termination-point=matr-1-0-2-http-c-ol-2-1-2-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/application-name]'
```

Comment:
As per the decision https://github.com/openBackhaul/ApplicationPattern/issues/714#issuecomment-1594314255, "find" needs to modified to "find or update, but not at OldRelease or NewRelease,"